ocp_cleanup.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Determine undercloud and allocate inventory
+  hosts: localhost
+  gather_facts: false
+  vars_files:
+    - vars/shift_stack_vars.yaml
+  tasks:
+    - name: list openstack projects for installed clusters
+      shell: |
+        ansible all -i '{{ undercloud_host }},' -m shell -a 'source /home/stack/overcloudrc && openstack project list -c Name -f value | grep rhocp' -u stack | grep rhocp
```

Comment:
> instead of using the adhoc command can you run this task on undercloud using the shell module

This task runs on jumphost.., running it on undercloud directly needs a manual intervention to prepare inventory file.
we already have undercloud_host[1] from shift_stack_vars.yaml
used adhoc command, avoiding to create a special inventory for this.
[1] https://github.com/redhat-performance/jetpack/pull/485/commits/f3a0f45abb2d5ed0f87832b9e25a17223adf42d3#diff-f9144ea1aa91b6992a70c42069fbd7414958b637270934198cbf3d5d2a25681aR10